### PR TITLE
list.sorted implemented

### PR DESCRIPTION
**Pull request by EugeneDar**
https://github.com/objectionary/eo-collections/issues/14

### CODE REVIEW

**Comment by EugeneDar** [comment]
@Graur please, have a look

File: src/test/eo/org/eolang/collections/list-tests.eo [hunk 1]
```diff
@@ -471,3 +471,53 @@
...
+
+[] > one-element-list-sorted
+  sorted. > res!
+    list
+      * 42
+  assert-that > @
+    res
+    $.equal-to
+      list
+        * 42
```
Comment (Graur) [inline src/test/eo/org/eolang/collections/list-tests.eo:523]: @EugeneDar please, add corner test cases with an empty array and with negative numbers

**Review by Graur — CHANGES_REQUESTED** [review]
@EugeneDar I like it, just a few comments from my side:

File: src/test/eo/org/eolang/collections/list-tests.eo (hunk 1 above)
Comment (EugeneDar) [reply src/test/eo/org/eolang/collections/list-tests.eo:523]: @Graur sure, I'll do it

**Comment by EugeneDar** [comment]
@Graur please, have a look again

**Review by Graur — APPROVED** [review]
@EugeneDar looks good to me, thanks!
@yegor256 please, have a look

**Comment by EugeneDar** [comment]
@yegor256 please, have a look

**Comment by yegor256** [comment]
@rultor merge
